cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -48,10 +49,13 @@ class SublandingFilterablePage(FilterableFeedPageMixin, FilterableListMixin, CFG
 
     template = 'sublanding-page/index.html'
 
+    objects = PageManager()
```

Comment:
Can `objects` live on CFGOVPage so this doesn't have to be defined for each page?